Library/Formula/pixz.rb

```diff
@@ -17,17 +18,22 @@ class Pixz < Formula
   end
 
   def install
+    ENV["PKG_CONFIG_PATH"] = "#{HOMEBREW_PREFIX}/lib/pkgconfig:#{HOMEBREW_PREFIX}/opt/libarchive/lib/pkgconfig"
```

Comment:
Is there a specific need to override the whole `PKG_CONFIG_PATH`?

Would rather do something like:

```ruby
ENV.prepend_path "PKG_CONFIG_PATH", Formula["libarchive"].opt_lib/"pkgconfig"
```
If it's having trouble finding that for whatever reason.